src/Apps/Artist/Routes/CV/CVItem.tsx

```diff
@@ -229,7 +229,7 @@ export const CVPaginationContainer = createPaginationContainer(
     getFragmentVariables(prevVars, totalCount) {
       return { ...prevVars, count: totalCount }
     },
-    getVariables(props, { count, cursor }, fragmentVariables) {
+    getVariables(props: CVItemProps, { count, cursor }, fragmentVariables) {
```

Comment:
This shouldn’t be necessary and may indicate an enhancement possibility in the react-relay typings.

Reply:
Yeah the types look weird. They have the props type available as a generic parameter but seems like they deliberatly decide to avoid it.

![image](https://user-images.githubusercontent.com/1242537/59272946-0672c280-8c4f-11e9-83ce-fc480f87627c.png)

Could also have just been overlooked while they were migrating from non-generic code?

Reply:
(They is probably me :D)

Reply:
Now that I mention it, I recall this is actually something that’s on my TODO list https://github.com/artsy/emission/blob/7a06739211a4545cdd56eed77a5619006cf9c722/patches/%40types/react-relay%2B1.3.9.patch